charts/apisix/templates/configmap.yaml

```diff
@@ -250,9 +252,26 @@ data:
     {{- end }}
 
     deployment:
-      role: traditional
+      role: {{ .Values.deployment.role }}
+      {{- if or (eq .Values.deployment.role "traditional") (eq .Values.deployment.role "control_plane") }}
```

Comment:
Should it not be  (eq .Values.deployment.mode"traditional") rather than  (eq .Values.deployment.role "traditional") ?

Reply:
no, Although traditional mode and role values are the same. But here, in order to maintain the unity of logic, we choose to judge the role.